### DISCUSSION TITLE
change the open menu bind

### DISCUSSION THREAD

**Discussion by Ksg02** [YimMenu/YimMenu#3116] [Q&A]
I set the bind to Lshift and \ but when i press the buttons it does not open can anyone help me reset the bind. i have also complete deleated and reinstalled the menu

**Comment by lonelybud** [comment]
delete this file - `"C:\Users\XXX\AppData\Roaming\YimMenu\settings.json"`.  insert key will be default then. when you inject next time then change to whatever again.

**Reply by Ksg02** [reply]
thank you so much. this was the solution. i was looking in the master file

**Reply by kraken307** [reply]
where do i find the file to delete?

**Reply by lonelybud** [reply]
to open yimmenu folder, go to start, type run and in run paste this 
`%userprofile%\AppData\Roaming\YimMenu`
this will open up the folder and you can manually delete the file `settings`

alternatively open cmd.exe as administrator and paste this command and press enter
`del %userprofile%\AppData\Roaming\YimMenu\settings.json`

then reinject menu and use insert key to toggle

**Comment by Dydax18** [comment]
merci beaucoup
